tools/bumper/git.go

```diff
@@ -0,0 +1,175 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"os/exec"
+	"strings"
+
+	"github.com/google/go-github/v32/github"
+	"github.com/pkg/errors"
+	"golang.org/x/oauth2"
+)
+
+// gitHubManager does the Github operations handling.
+type gitHubManager struct {
+	// github client.
+	client *github.Client
+
+	// context needed for github api
+	ctx context.Context
+}
+
+func NewGitHubManager(token string) (*gitHubManager, error) {
+	ctx := context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: token},
+	)
+	tc := oauth2.NewClient(ctx, ts)
+
+	githubMgr := &gitHubManager{
+		client: github.NewClient(tc),
+		ctx:    ctx,
+	}
+
+	err := githubMgr.verifyClient()
+	if err != nil {
+		return nil, errors.Wrap(err, "Failed to verify connection with github client API")
+	}
+
+	return githubMgr, nil
+}
+
+func (mgr *gitHubManager) verifyClient() error {
+	//do a small sanity check
```

Comment:
This comment on its own is quite useless. Describe what you are doing in more detail or make the function self explanatory by its name.

Reply:
Done